### ISSUE TITLE
obscure error if jwtSecret isn't configured

### ISSUE THREAD

**Issue by pekeler** [arangodb/foxx-sessions-jwt#3]
I think it would be good if this app threw an explicit error telling me that this value is missing, instead of this obscure stack trace: https://github.com/arangodb/arangodb/issues/1696

**Comment by pluma** [comment]
Indeed. The problem is that the secret is not required because the "none" type encryption must not use a secret whereas every other type must use a secret. Therefore we can't rely on the app not being mounted unless properly configured and instead need to catch misconfiguration at runtime.
